celery/canvas.py.1

```diff
@@ -0,0 +1,1806 @@
+"""Composing task work-flows.
+
+.. seealso:
+
```

Comment:
curious what this file doing? just renamed?

Reply:
Oops, this shouldn't have been included.  It's should be gone now.